api/cloudcontroller/ccv3/client.go

```diff
@@ -7,7 +7,7 @@
 // may include features and endpoints of later API versions.
 //
 // For more information on the Cloud Controller API see
-// https://apidocs.cloudfoundry.org/
+// https://v2-apidocs.cloudfoundry.org/
```

Comment:
This is v3 client. Do we want to make changes here?

Reply:
you're right--we don't want this to be v2. However, since this is the v3 client and not a "latest client", changing the link to v3 specifically